adding fall 2023 events and header images (#617)
CHTC/chtc-website-source: _events/2023-09-27-information-session.md

---
short_title: CHTC Information Session
title: "CHTC Information Session"
published: true

excerpt: |
    Want to learn more about how the CHTC could support your research? Come to an information session (with pizza!) on September 27. 

start_date: 2023-09-27
end_date: 2023-09-27
location: "UW-Madison Campus, Room 1240 of the Computer Sciences Building"
image: "/images/CDIS_Career_Fair22_3942.jpg"
header_image: "/images/CDIS_Career_Fair22_3942.jpg"
---

{% capture main %}

<p style="font-size: larger; font-weight: bold;">Learn about CHTC's computing 
and data services for UW-Madison affiliates at this information session!</p>

Come eat pizza and meet CHTC's Research Computing Facilitation team on 
Wednesday, September 27. We'll present a general introduction to CHTC services, 

> with time to reflect and discuss how your research might fit onto CHTC resources. 

Reviewer: ```suggestion
with time to reflect and discuss how your research might fit the resources offered by the Center for High Throughput Computing.
```

{% endcapture %}

{% capture subsection %}
{: .mt-0 }
### Who

> CHTC Interested Users

Reviewer 2: ```suggestion
Researchers, students, staff, and faculty interested in using CHTC resources
```

> CHTC Interested Users

Reviewer: ```suggestion
UW-Madison students and staff interested in large-scale computing
```

### When

Wednesday, September 27, 12:00 - 1:00 pm CST

### Where

> UW-Madison Campus, Room 1240, Computer Sciences Building

Reviewer: ```suggestion
Computer Sciences Building, Room 1240, UW-Madison Campus
```
